Source/WTF/wtf/PlatformHave.h

```diff
@@ -1908,6 +1908,7 @@
     || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MAX_ALLOWED >= 120000) \
     || (PLATFORM(VISION) && __VISION_OS_VERSION_MIN_REQUIRED >= 20000) \
     || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 180000) \
-    || (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 150000))
+    || (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 150000) \
+    || !defined(__APPLE__))
```

Comment:
`!PLATFORM(COCOA)` is more common I think.

Reply:
Fixed. Thank you for the review.